### ISSUE TITLE
Error with plugin in .babelrc

### ISSUE THREAD

**Issue by KateKate** [megawac/babel-plugin-ramda#11]
The last release seems to have a conflict with babel. I've got the following error after `npm i`:
`Error: Plugin 0 specified in "path/to/project/.babelrc" provided an invalid property of "F"`

My .babelrc
```
{
  "presets": ["react", "es2015", "stage-0"],
  "plugins": ["ramda"]
}
```

Thank you!

**Comment by megawac** [comment]
@KateKate my bad, I didn't build before publishing. It should be fixed now

**Comment by KateKate** [comment]
Thank you! That one is fixed. The new one is appeared:
Cannot read property 'file' of undefined
    at PluginPass.Identifier (/path/to/project/node_modules/babel-plugin-ramda/lib/index.js:81:23)

**Comment by megawac** [comment]
Interesting. The `index.js` file looks fine to me locally. Do you mind providing a snippet of code where the plugin fails on?

**Comment by KateKate** [comment]
oops, it's not error with your package. it's another story ) sorry.

**Comment by mAAdhaTTah** [comment]
I'm getting the issue as originally posted as of 1.1.4:

```
Error: Plugin 0 specified in "path/to/project/.babelrc" provided an invalid property of "F"
```

**Comment by megawac** [comment]
Whoops, forgot to rebuild the plugin

**Comment by megawac** [comment]
done

**Comment by MikeLP** [comment]
Reproduced again

**Comment by megawac** [comment]
My bad will fix

**Comment by megawac** [comment]
Should be fixed in 1.1.9
